app/views/News.js

```diff
@@ -33,11 +33,13 @@ class NewsScreen extends Component {
       name: 'Safe Paths News', // TODO: translate
       url: 'https://privatekit.mit.edu',
     };
+
     this.state = {
       visible: true,
       default_news: default_news,
       newsUrls: [default_news, default_news],
       current_page: 0,
+      iscached: false,
```

Comment:
follow other key_name_convention. Also, `is_cached` is unclear which cache it's referring to. 

Perhaps `enable_webview_cache` is more clear?

Reply:
Changed variable name